src/app/pages/acc-managements/acc-management/acc-management.component.ts

```diff
@@ -73,14 +82,30 @@ export class AccManagementComponent implements OnInit {
   updateSuccess($event) {
     this.editedUser = $event;
     this.editingUsername = null;
+    this.managementUsersService.getUser().subscribe(res => {
+      this.listUser = <Array<User>>res.data;
+      // add isEdited for all item
+      this.listUser.forEach(function (user, userIndex) {
```

Comment:
move to new function and call it to avoid duplicate code

Reply:
done